package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-loader",
-  "version": "6.1.0",
+  "version": "6.1.1",
```

Comment:
you shouldn't set the version in the `package.json` in PRs. let the maintainers do that